recipe/meta.yaml

```diff
@@ -15,6 +15,7 @@ source:
 
 build:
   number: 0
+  skip: True  # [py2k]
```

Comment:
Version 0.6.5 doesn't actually support Python 2 either. It uses a py3 only feature of tempfile: https://github.com/maxpoint/conda-mirror/blob/master/conda_mirror/conda_mirror.py#L510